src/connect/set-contracts.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+function setContracts(networkID, allContracts) {
```

Comment:
Actually, this (and other such instances) should also be changed to `networkId` to follow the convention used elsewhere.

Reply:
Yeah, I realize it's not possible to update every instance. Might as well update as many as we can, though.